include/wlr/render/egl.h

```diff
@@ -16,10 +16,13 @@
 #define EGL_NO_X11
 #endif
 
+#include <wlr/config.h>
+
 #include <EGL/egl.h>
 #include <EGL/eglext.h>
-// TODO: remove eglmesaext.h
```

Comment:
This TODO should be retained. Including `eglmesaext.h` is still a hack.

Reply:
I added it back. Thanks.